app/javascript/gobierto_visualizations/webapp/containers/contracts/ContractsShowTable.vue

```diff
@@ -0,0 +1,53 @@
+<template>
```

Comment:
Pay attention to the filepaths. This is an individual component, so it shouldn't be under `containers`, but `components`

Reply:
Done!